File: lab/lab07/count_word.pl
#!/usr/bin/perl -w
#print"$ARGV[0],,,,,,,$ARGV[1]";

$aim=$ARGV[0];
$num=0;
foreach $line (<STDIN>){
	$line=lc $line;
	@words=($line=~/[a-z]+/g);
	foreach $word (@words){
		if($word eq $aim){$num++;}
	}
}

print"$aim occurred $num times\n";

File: lab/lab07/count_word.py
import sys
import re

target_word = sys.argv[1]
#print(target_word)
count_number = 0
for line in sys.stdin:
    # print(line)
    for i in re.findall('[a-z]+', line):
        if i == target_word:
            count_number += 1
print(target_word+" occurred "+str(count_number)+" times")
